compositorcontroller.cpp

```diff
@@ -1757,6 +1768,11 @@ namespace RdkShell
                 gTopmostCompositor = gCompositorList.front();
             }
         }
+        else
+        {
+            gTopmostCompositor.name = "";
+            gTopmostCompositor.compositor = nullptr;
+        }
```

Comment:
I think we only want to clear the topmost compositor if the client passed in is currently the topmost compositor and topmost is false.